ios/FluentUI/Date Time Pickers/Date Time Picker/Views/DateTimePickerView.swift

```diff
@@ -153,15 +153,15 @@ class DateTimePickerView: UIControl {
             x += viewWidth
         }
 
-        let lineOffset = round((frame.height - DateTimePickerViewComponentCell.idealHeight - 2 * selectionTopSeparator.frame.height) / 2)
+        let lineOffset = round((frame.height - DateTimePickerViewComponentCell.idealHeight - 2 * selectionTopDivider.view.frame.height) / 2)
```

Comment:
Can you store selectionTopDivider.view in a constant to be used throughout this implementation?